.theia/settings.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Curious - is this file doing?

Reply:
I'm not sure, don't remember making it...